docs/tasks/argo.rst

```diff
@@ -0,0 +1,283 @@
+  ServiceAccount in production. You must create a more defined RBAC to grant access for
+  allowing creation of Custom Resources (ActionSets) in the ``Kanister`` namespace.
+
+Step 5 - Launching the Cron Workflow
+------------------------------------
+
+Let's launch the workflow in the ``argo`` namespace by running -
+
+.. code-block:: bash
+
+  argo cron create mysql-cron-wf.yaml -n argo
+
+Check if the workflow was created by running -
+
+.. code-block:: bash
+
+  argo cron list -n argo
+
+When the workflow runs, check if the ActionSet was created in the ``kanister`` namespace.
+
+.. code-block:: bash
+
+  kubectl get actionsets.cr.kanister.io -n kanister
+
+The output will be as follows.
+
+.. code-block:: bash
+
+  $ argo cron create mysql-cron-wf.yaml -n argo
+  > Name:                   mysql-cron-wf
+    Namespace:              argo
+    Created:                Fri Jul 22 10:23:09 -0400 (now)
+    Schedule:               */5 * * * *
+    Suspended:              false
+    ConcurrencyPolicy:      Replace
+    NextScheduledTime:      Fri Jul 22 10:25:00 -0400 (1 minute from now) (assumes workflow-controller is in UTC)
+
+  $ argo cron list -n argo
+  > NAME            AGE    LAST RUN    NEXT RUN      SCHEDULE      TIMEZONE        SUSPENDED
+    mysql-cron-wf   12s    N/A         1m            */5 * * * *                   false
+
+  $ argo cron list -n argo
+  > NAME            AGE    LAST RUN    NEXT RUN      SCHEDULE      TIMEZONE        SUSPENDED
+    mysql-cron-wf   4m     2m          2m            */5 * * * *                   false
+
+  $ kubectl get actionsets.cr.kanister.io -n kanister
+  > NAME            AGE
+    backup-478lk    2m28s
+
+Here, the workflow was created and scheduled to run in 1 minute.
+The time can be anywhere between the first 5 minutes for you. After it ran
+successfully, the `last run` field was updated with the timestamp of the last run.
+A backup ActionSet was created. The age of this ActionSet is seen as recently created.
+
+Submit the Cron Workflow by running -
+
+.. code-block:: bash
+
+  argo submit mysql-cron-wf.yaml
+
+You should see the submitted workflow on the Argo UI -
+
+.. image:: img/argo-cron-created-ui-list.png
+
+On clicking on the workflow name, you will see its status -
+
+.. image:: img/argo-cron-created-ui-desc.png
+
+Troubleshooting
+===============
+
+If the Cron Workflow does not run, check if the pod to run the workflow was
+created in the ``argo`` namespace. Examine the logs of this pod.
+
+.. code-block:: bash
+
+  kubectl logs [NAME_OF_MYSQL_CRON_WORKFLOW_POD] -n argo
+
+If this pod was not created, examine the logs of the Argo Workflow Controller
+in the ``argo`` namespace.
+
+.. code-block:: bash
+
+  kubectl logs [NAME_OF_WORKFLOW_CONTROLLER] -n argo
+
+If the logs mention that you have not granted the right permissions to the
+ServiceAccounts, circle back to Step 4 and check the names of your ServiceAccounts
+and namespaces.
```

Comment:
Not sure I understand the namespaces part here. Should we specify all the commands they should run?

Reply:
Step 4 was changed recently and there would be a lot more items to verify than just the ServiceAccounts and namespaces. A better phrasing would be
```
If the logs mention that you have not granted the right permissions to the
ServiceAccounts, circle back to Step 4 and verify your RBAC configuration.
Your ServiceAccount should have access to the requested resources.
```

Reply:
The specific commands are all present in Step 4 for them to configure RBAC.
Changed this phrasing in the doc.

Reply:
Sounds better now 👍🏼